### PR TITLE
refactor: upgrade elasticsearch image from 8.8.2 to 8.9.2

### PR DESCRIPTION
**Pull request by HamzaMasood1**
### What's in this PR?
Issue: https://github.com/camunda/distribution/issues/140

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [ ] In the repo's root dir, run `make go.update-golden-only`.
- [ ] There is no other open [pull request](../pulls) for the same update/change.
- [ ] Tests for charts are added (if needed).
- [ ] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [ ] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Comment by HamzaMasood1** [comment]
Should I also take this opportunity to update the chart version as well?

**Comment by HamzaMasood1** [comment]
I have installed the current release locally with the new es version. 
I have also upgraded locally using the new es version. 
Both are working fine for me.

**Review by aabouzaid — CHANGES_REQUESTED** [review]
We need to change it in renovatebot config too.

https://github.com/camunda/camunda-platform-helm/blob/d1f62349ead5a7f92bb80b7bbdaa24d28123e721/.github/renovate.json5#L57

File: .github/renovate.json5 [hunk 1]
```diff
@@ -54,12 +54,12 @@
       "allowedVersions": "~0.8.0"
     },
     {
-      // Limit Elasticsearch version to latest supported version in Camunda v8.3.
+      // Limit Elasticsearch version to latest supported version in Camunda v8.9.
```
Comment (aabouzaid) [inline .github/renovate.json5:57]: That's the Camunda version :grin: 
It should be 8.4 in that case.

Comment (HamzaMasood1) [reply .github/renovate.json5:57]: 🤦

**Pull request merged**